apps/spruce/cypress/integration/projectSettings/github_app_settings.ts

```diff
@@ -58,7 +58,7 @@ describe("GitHub app settings", () => {
     cy.get("@appId").should("have.value", "12345");
     cy.get("@privateKey").should("have.value", "{REDACTED}");
     cy.get("@appId").should("have.attr", "aria-disabled", "true");
-    cy.get("@privateKey").should("have.attr", "aria-disabled", "true");
+    cy.get("@privateKey").should("have.attr", "disabled", "");
```

Comment:
I think maybe the test is failing because you want to call it like this:
```suggestion
    cy.get("@privateKey").should("have.attr", "disabled");
```